suite/tests/api/ir_x86.c

```diff
@@ -299,6 +299,88 @@ test_all_opcodes_3_avx512_vex(void *dc)
 #    undef INCLUDE_NAME
 }
 
+static void
+test_all_opcodes_mask_2_avx512_evex(void *dc)
+{
+#    define INCLUDE_NAME "ir_x86_mask_2args_avx512_evex.h"
+#    define OPCODE_FOR_CREATE(name, opc, icnm, flags, arg1, arg2, arg3)             \
+        do {                                                                        \
+            if ((flags & IF_X64_ELSE(X86_ONLY, X64_ONLY)) == 0) {                   \
+                instrlist_append(ilist, INSTR_CREATE_##icnm(dc, arg1, arg2, arg3)); \
+                len_##name = instr_length(dc, instrlist_last(ilist));               \
+            }                                                                       \
+        } while (0);
+#    include "ir_x86_all_opc.h"
+#    undef OPCODE_FOR_CREATE
+#    undef INCLUDE_NAME
+}
+
+static void
+test_disas_mask_2_avx512_evex(void *dc)
```

Comment:
This should be down below rather in the middle of the ir_x86_all_opc tests.

Reply:
Done.